projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1227,12 +1237,65 @@ function wpcom_launchpad_has_translation( $string, $domain = 'jetpack-mu-wpcom'
 	return $translated_string !== $string;
 }
 
+/**
+ * Determine `add_new_page` task visibility. The task is visible if there is no 'About' page on the site.
+ *
+ * @return bool True if we should show the task, false otherwise.
+ */
+function wpcom_launchpad_is_add_about_page_visible() {
+	return ! wpcom_is_update_about_page_task_visible();
+}
+
+/**
+ * Completion hook for the `add_about_page` task.
+ *
+ * @param int    $post_id The post ID.
+ * @param object $post    The post object.
+ */
+function wpcom_launchpad_add_about_page_check( $post_id, $post ) {
+	// Ensure that Headstart posts don't mark this as complete
+	if ( defined( 'HEADSTART' ) && HEADSTART ) {
+		return;
+	}
+
+	// We only care about pages, ignore other post types.
+	if ( $post->post_type !== 'page' ) {
+		return;
+	}
+
+	// We only care about published pages. Pages added via the API are not published by default.
+	if ( $post->post_status !== 'publish' ) {
+		return;
+	}
+
+	// Don't do anything if the task is already complete.
+	if ( wpcom_is_task_option_completed( array( 'id' => 'add_about_page' ) ) ) {
+		return;
+	}
+
+	// If the page is the previously located About page, ignore it.
+	$about_page_id = wpcom_get_site_about_page_id();
+	if ( null !== $about_page_id && $post->ID === $about_page_id ) {
+		return;
+	}
+
+	$is_about_page = 'about' === get_post_meta( $post->ID, '_wp_layout_category', true );
+	if ( ! $is_about_page ) {
+		return;
+	}
```

Comment:
I think we need to confirm the data model/structure and naming for the layout category meta field before we can ship this PR. I _think_ we want to use a `_wpcom_` prefix, and I also think we should store an array of category values.

Separately, we also need to confirm that the post_meta key is registered -- it's possible that the Editor Toolkit plugin is disabled on some Atomic sites.
```suggestion
	// Don't do anything if we don't have the page category post_meta field registered.
	if ( ! registered_meta_key_exists( 'post', '_wpcom_layout_category', 'page' ) ) {
		return;
	}

	// Don't do anything if the task is already complete.
	if ( wpcom_is_task_option_completed( array( 'id' => 'add_about_page' ) ) ) {
		return;
	}

	// If the page is the previously located About page, ignore it.
	$about_page_id = wpcom_get_site_about_page_id();
	if ( null !== $about_page_id && $post->ID === $about_page_id ) {
		return;
	}

	$stored_page_categories = get_post_meta( $post->ID, '_wpcom_page_layout_category', true );
	if ( ! is_array( $stored_page_categories ) || ! in_array( 'about', $stored_page_categories, true ) ) {
		return;
	}
```